py/desisim/templates.py

```diff
@@ -358,7 +358,7 @@ def _blurmatrix(self, vdisp, log=None):
         blurmatrix = dict()
         for uvv in uvdisp:
             sigma = 1.0 + (self.basewave * uvv / LIGHT)
-            blurmatrix[uvv] = pxs.gauss_blur_matrix(self.pixbound, sigma)
+            blurmatrix[uvv] = pxs.gauss_blur_matrix(self.pixbound, sigma).astype('f4')
```

Comment:
I'm not convinced that keeping the blur matrix as f4 helps, since it is likely upcast back to f8 every time it is used, perhaps creating even more memory churn since an f8 version would be created multiple times and the garbage collector may or may not keep up.  Are you sure this helps?